dev-docs/publisher-api-reference.md

```diff
@@ -1474,14 +1467,38 @@ Additional information of these properties:
 | `endpoint` | Required | URL | Defines the auction endpoint for the Prebid Server cluster |
 | `syncEndpoint` | Required | URL | Defines the cookie_sync endpoint for the Prebid Server cluster |
 | `userSyncLimit` | Optional | Integer | Max number of userSync URLs that can be executed by Prebid Server cookie_sync per request.  If not defined, PBS will execute all userSync URLs included in the request. |
-| `adapterOptions` | Optional | Object | Arguments will be added to resulting OpenRTB payload to Prebid Server. |
+| `adapterOptions` | Optional | Object | Arguments will be added to resulting OpenRTB payload to Prebid Server in request.ext.BIDDER. See the example above. |
+| `extPrebid` | Optional | Object | Arguments will be added to resulting OpenRTB payload to Prebid Server in ext.prebid. See video-related example below. |
 
-**Additional Notes on s2sConfig properties**
+**Notes on s2sConfig properties**
 
 - Currently supported vendors are: appnexus & rubicon
 - When using `defaultVendor` option, `accountId` and `bidders` properties still need to be defined.
 - If the `s2sConfig` timeout is greater than the Prebid.js timeout, the `s2sConfig` timeout will be automatically adjusted to 75% of the Prebid.js timeout in order to fit within the auction process.
 
+**Video via s2sConfig**
```

Comment:
Do we also want to mention that `extPrebid` can be used for working with mobile display ads as well?

Can enable mobile cache by extPrebid.cache.bids = {}; right?

Reply:
I don't think mentioning mobile is necessary because only the SDK needs to cache bids, and that doesn't utilized s2sConfig.